src/Symfony/Component/DependencyInjection/Loader/FileLoader.php

```diff
@@ -139,13 +148,20 @@ private function findClasses($namespace, $pattern, $excludePattern)
             if (!preg_match('/^[a-zA-Z_\x7f-\xff][a-zA-Z0-9_\x7f-\xff]*+(?:\\\\[a-zA-Z_\x7f-\xff][a-zA-Z0-9_\x7f-\xff]*+)*+$/', $class)) {
                 continue;
             }
+
+            try {
+                $r = $this->container->getReflectionClass($class);
+            } catch (\ReflectionException $e) {
+                $classes[$class] = $e->getMessage();
```

Comment:
It's a little awkward to return an array with the error as the key, but since this method is private, it seems safe.

Reply:
Actually, the class is the key here. I agree that the structure I'm using now is a bit more awkward than the old one, but as you said, the method is private, so it's a purely internal data structure.

That being said, if you have a suggestion regarding a better data structure, I'm all ears. 😃